shell/client/grain-client.js

```diff
@@ -1509,12 +1523,18 @@ Meteor.startup(function () {
         const renderedTemplate = template.replace(/\$API_TOKEN/g, tokenId)
                                          .replace(/\$API_HOST/g, host)
                                          .replace(/\$GRAIN_TITLE_SLUG/g, grainTitleSlug);
+        let link = undefined;
+        if (rawLink) {
+          link = `${rawLink.scheme}://${window.location.protocol}//${host}#${tokenId}`;
```

Comment:
Change the first `://` to just `:` -- the `//` doesn't make sense since it's not followed by a hostname.